---
layout: page
---
# About

## HuBMAP Consortium
The HuBMAP Consortium develops tools to integrate & map diverse biological data across the healthy human body. HuBMAP tools and data are freely & openly available to the global research community. Our aim is to accelerate efforts to understand the relationships between the organization of cells and tissues, biological function and human health.

### Consortium Data Access Policies & Procedures
[Policies](https://hubmapconsortium.org/policies/)

### Consortium Data Use Agreement (DUA)
[Data Use Agreement](https://hubmapconsortium.org/wp-content/uploads/2020/06/DUA_FINAL_2020_02_03_for_Signature.pdf) (as of 3 February 2020)

### Protected Data & Access via dbGaP
[Access dbGaP](https://dbgap.ncbi.nlm.nih.gov/aa/wga.cgi?page=login) for HuBMAP protected data - please bring along study ID

### Citation
Investigators using HuBMAP data in publications or presentations are requested to cite _The Human Body at Cellular Resolution: the NIH Human BioMolecular Atlas Program_ (doi:[10.1038/s41586-019-1629-x](https://doi.org/10.1038/s41586-019-1629-x)) and to include an acknowledgement of HuBMAP. Suggested language for such an acknowledgment is: “The results here are in whole or part based upon data generated by the [NIH Human BioMolecular Atlas Program](https://hubmapconsortium.org) (HuBMAP).

### Funding
- [Common Fund of the National Institutes of Health](https://commonfund.nih.gov/HuBMAP)
- [Funded Research](https://commonfund.nih.gov/HuBMAP/fundedresearch)
 
### Contact
General - help@hubmapconsortium.org

Public Relations/Media - [Robin Scibek](mailto:flaus@psc.edu) (Pittsburgh Supercomputing Center)

NIH Program - [Richard Conroy](mailto:richard.conroy@nih.gov) (NIH Common Fund)

### Team

<table>
<thead>
  <tr>
    <th>Site</th>
    <th>Team</th>
    <th>Contributions</th>
  </tr>
</thead>
<tbody>
<tr>
  <td>
  Carnegie Mellon University<br>
  PI: Ziv Bar-Joseph<br>
  </td>
  <td>
  Matt Ruffalo<br>
  Maria Keays<br>
  Vasyl Vaskivsky<br> 
  Sean Donahue<br>
  Ted Zhang<br>
  Walt Shands<br>
  Allyson Ricarte<br>
  Vladimir Kiselev <br>
  Robert Murphy<br>
  Sarah Teichmann <br>
  Ben Paten<br>
  </td>
  <td>
  Pipelines for sc and bulk RNA-Seq<br>
  Pipelines for sc and bulk ATAC-Seq<br>
  Pipelines for codex<br>
  Pipelines for Microscopy and Mass-Spec data<br>
  SPRM for analysis of imagining data<br>
  </td>
</tr>
<tr>
  <td>
  Harvard Medical School<br>
  PI: <a href="http://gehlenborglab.org/team/members/nils-gehlenborg/">Nils Gehlenborg</a><br>
  </td>
  <td>
  Christine Briggs<br>
  John Conroy<br>
  Chuck McCallum<br>
  Ilan Gold<br>
  Mark Keller<br>
  Tiffany Liaw<br>
  Trevor Manz<br>
  Margaret Vella<br>
  </td>
  <td>
  User Interfaces<br>
  Data Visualization<br>
  Curation<br>
  Metadata<br>
  </td>
</tr>
<tr>
  <td>
  Indiana University<br>
  PI: <a href="http://cns.iu.edu">Katy Börner</a><br>
  </td>
  <td>
  Daniel Bolin<br>
  Kristen Browne (NIAID)<br>
  Andreas D Bueckle<br>
  Leonard E Cross<br>
  Bruce W Herr II<br>
  Edward Lu<br>
  Matthew S Martindale<br>
  Mark A Musen (Stanford U)<br>
  Hrishikesh Paul<br>
  Adam Phillips<br>
  Ellen M Quardokus<br>
  Elizabeth Record<br>
  Griffin M Weber (HMS)<br>
  Yingnan Yu<br>
  </td>
  <td>
  Common Coordinate Framework (CCF)<br>
  CCF Ontology<br>
  ASCT+B Table Visualizations<br>
  Registration User Interface<br>
  Exploration User Interface<br>
  3D Object Library<br>
  Visible Human MOOC<br>
  </td>
</tr>
<tr>
  <td>
  University of Pittsburgh<br>
  PI: <a href="https://rio.pitt.edu">Jonathan Silverstein<a><br>
  </td>
  <td>
  Becky Boes<br>
  Chuck Borromeo<br>
  Peter Kant<br>
  Desheng Li<br>
  Melissa Schwenk<br>
  Bill Shirey<br>
  Zhou (Joe) Yuan<br>
  </td>
  <td>
  Dev Ops<br>
  APIs and Gateway<br>
  Provenance<br>
  Data Ingest<br>
  Data release coordination<br>
  Documentation<br>
  Security<br>
  </td>
</tr>
<tr>
  <td>
  Pittsburgh Supercomputing Center<br>
  PI: Philip Blood<br>
  </td>
  <td>
  Philip Blood<br>
  Ivan Cao-Berg<br>
  Kenneth Chiacchia<br>
  Stephen Deems<br>
  Shane Filus<br>
  Deb Nigra<br>
  Alex Ropelewski<br>
  Robin Scibek<br>
  Derek Simmel<br>
  Julian Uran<br>
  Joel Welling<br>
  </td>
  <td>
  Hybrid Cloud Infrastructure<br>
  Workflow Orchestration<br>
  Data Ingest<br>
  Data Management<br>
  Metadata<br>
  Security<br>
  Help Desk<br>
  Communications<br>
  Coordination<br>
  Policy<br>
  </td>
</tr>
<tr>
  <td>
  New York Genome Center / EBI<br>
  PIs: Rahul Satija, John Marioni<br>
  </td>
  <td>
  Jaison Jain<br>
  Andrew Butler<br>
  Tim Stuart<br>
  Bill Mauck<br>
  Aviv Regev<br>
  Tommasso Biancalini<br>
  Alsu Missarova<br>
  Shila Ghazanfar<br>
  </td>
  <td>
  Data Harmonization<br>
  Data Integration<br>
  Cell Annotation<br>
  </td>
</tr>
</tbody>
</table>


### Software
HuBMAP [GitHub](https://github.com/hubmapconsortium) Organization
